base/inc/attachments.php

```diff
@@ -32,9 +32,13 @@ function siteorigin_widgets_get_attachment_image_src( $attachment, $size, $fallb
 	return false;
 }
 
-function siteorigin_widgets_get_attachment_image( $attachment, $size, $fallback ){
+function siteorigin_widgets_get_attachment_image( $attachment, $size, $fallback, $class = false ){
```

Comment:
We should really keep the arguments of `siteorigin_widgets_get_attachment_image` the same as `wp_get_attachment_image`. So rather than having a `$class` argument, it should be a general `$atts` that's processed and passed through to `wp_get_attachment_image`.